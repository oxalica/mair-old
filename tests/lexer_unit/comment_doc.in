     /**/
a/* */b
a// /*b
a//\nb
a/*/**/*/a
a/*/**//*/**/*/*/a
a/*/*/*/*/**/*/*/*/*/a
a/*0/**/"/*'/*/ */*/#*/ a
/*
/*/**/*//**
/*/*! */
a/*0/**/"/*'/*//*/+*/#*/ a
-/*-*/-*/-
////
//// a
/***/
/****/
/*** */
///
///a\nb
//!\n
//! x
/*! a */
/** */
/*!*/
