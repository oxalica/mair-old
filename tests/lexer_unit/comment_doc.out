Ok([])
Ok([(Ident("a"), 0..1), (Ident("b"), 6..7)])
Ok([(Ident("a"), 0..1)])
Ok([(Ident("a"), 0..1)])
Ok([(Ident("a"), 0..1), (Ident("a"), 9..10)])
Ok([(Ident("a"), 0..1), (Ident("a"), 17..18)])
Ok([(Ident("a"), 0..1), (Ident("a"), 21..22)])
Ok([(Ident("a"), 0..1), (Ident("a"), 24..25)])
Err(LexicalError { pos: 0, kind: UnclosedComment })
Err(LexicalError { pos: 8, kind: UnclosedComment })
Err(LexicalError { pos: 0, kind: UnclosedComment })
Err(LexicalError { pos: 1, kind: UnclosedComment })
Ok([(Symbol(Sub), 0..1), (Symbol(Sub), 6..7), (Symbol(Mul), 7..8), (Symbol(Div), 8..9), (Symbol(Sub), 9..10)])
Ok([])
Ok([])
Ok([])
Ok([])
Ok([])
Ok([(OuterDoc(""), 0..3)])
Ok([(OuterDoc("a\\nb"), 0..7)])
Ok([(InnerDoc("\\n"), 0..5)])
Ok([(InnerDoc(" x"), 0..5)])
Ok([(InnerDoc(" a "), 0..8)])
Ok([(OuterDoc(" "), 0..6)])
Ok([(InnerDoc(""), 0..5)])
